### PR TITLE
Move topics to enum class

### PR DESCRIPTION
**Pull request by m4ra**
**Tasks**
- [ ] PR name contains story or task reference
- [x] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [x] Changelog

### CODE REVIEW

File: meinberlin/apps/contrib/enums.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    )
+    URB = (
+        "URB",
+        _("Urban development"),
+    )
+    ENV = (
+        "ENV",
+        _("Environment & public green space"),
+    )
+    TRA = "TRA", _("Traffic")
```
[the remark this answers is not in the document]
Comment (m4ra) [reply meinberlin/apps/contrib/enums.py:22]: found it. it was black that was changing it. Is there a reason why we have both black and flake8? @goapunk

File: meinberlin/apps/plans/models.py [hunk 2]
```diff
@@ -196,9 +196,7 @@ def administrative_district(self):
 
     @property
     def topic_names(self):
-        if hasattr(settings, "A4_PROJECT_TOPICS"):
-            return [topic.name for topic in self.topics.all()]
-        return []
+        return [topic.__str__() for topic in self.topics.all()]
```
Comment (m4ra) [inline meinberlin/apps/plans/models.py:199]: @goapunk I will change these to str(topic) so we don't call private functions. and the __str__ one is specifically for the output of the instance when called with print() and str().

Comment (goapunk) [reply meinberlin/apps/plans/models.py:199]: @m4ra yes, good idea

**Review by goapunk — APPROVED** [review]
LGTM!

**Pull request merged**